### DISCUSSION TITLE
Examples of your Tabler work?

### DISCUSSION THREAD

**Discussion by robertandrews** [tabler/tabler#912] [Show and tell]
Post a showcase of your Tabler implementation?

**Comment by woto** [comment]
It is my pet project website https://goodreviews.ru 
I wish it will be available when dear stranger will visit it in the future ))
More about it [here](https://vc.ru/tribuna/282067-good-reviews-cpa-platforma-dlya-kopirayterov).

**Reply by woto** [reply]
Temporarily closed huh :) I have to come up with a new hypothesis.

**Comment by nicodemuz** [comment]
https://www.fruitsandfasting.com/

![image](https://github.com/tabler/tabler/assets/1620454/1bca00a5-77eb-4c29-86b4-131f91fc5e7b)

**Comment by nicodemuz** [comment]
https://www.artistic.wtf/ - A free text to image generator service

![image](https://github.com/tabler/tabler/assets/1620454/607f88c8-4f2c-4a04-b6c3-f504c8163a32)

![image](https://github.com/tabler/tabler/assets/1620454/d1e13271-41a7-47a2-9d79-464a7fcf2357)

**Comment by nicodemuz** [comment]
https://www.lifextenders.com/ - a community focusing on health and longevity

![image](https://github.com/tabler/tabler/assets/1620454/ec8db101-dcef-4fb4-9771-9998e634b842)
